Gemfile

```diff
@@ -50,6 +52,7 @@ group :development do
   gem 'rails_layout'
   gem 'spring-commands-rspec'
   gem 'ruby_parser'
+  gem 'puma'
```

Comment:
With webrick we have an error:
```
[HOMS] I, [2020-01-10T11:20:58.158937 #13583]  INFO -- : Started GET "/cable/" [WebSocket] for 127.0.0.1 at 2020-01-10 11:20:58 +0300
[HOMS] I, [2020-01-10T11:20:58.159043 #13583]  INFO -- : Successfully upgraded to WebSocket (REQUEST_METHOD: GET, HTTP_CONNECTION: Upgrade, HTTP_UPGRADE: websocket)
[HOMS] F, [2020-01-10T11:20:58.162750 #13583] FATAL -- :
NotImplementedError - only partial hijack is supported.:

127.0.0.1 - - [10/Jan/2020:11:20:52 MSK] "GET /cable HTTP/1.1" 500 5145
- -> /cable
```
[We have to use puma](https://stackoverflow.com/questions/36552041/notimplementederror-only-partial-hijack-is-supported-using-message-bus), it work